### PR TITLE
Fixes to outdated links/bad grammar in Special Thanks

### PR DESCRIPTION
**Pull request by Xeinok-Pals**

### CODE REVIEW

File: docs/guidelines/users/global-leaderboard-and-achievement-hunting-rules.md [hunk 1]
```diff
@@ -217,6 +217,6 @@ In cases where there is any more complex scripting than repeating the same butto
 
 ## Special Thanks
 
-This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://100pals.co/rules). This version is trimmed down and tailored with topics that are relevant to the RetroAchievements context.
+This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://achievementhunting.com/rules) (the "UAHR"). This version is trimmed down and tailored to topics that are specifically relevant to the unique RetroAchievements context, although additional details may be gleaned from the UAHR when needed.
```
Comment (wescopeland) [inline docs/guidelines/users/global-leaderboard-and-achievement-hunting-rules.md]:
```suggestion
This document is heavily based on the [100Pals Unified Achievement Hunting Rules](https://achievementhunting.com/rules) (the "UAHR"). This version is trimmed down and tailored to topics that are specifically relevant to the RetroAchievements context.
```
Minor adjustments. We don't want users pointing to an off-site document trying to find loopholes when they're untracked.

File: docs/guidelines/users/global-leaderboard-and-achievement-hunting-rules.md [hunk 2]
```diff
@@ -217,6 +217,6 @@ In cases where there is any more complex scripting than repeating the same butto
...
 
-The [100Pals](https://100pals.co/) is a community with experienced achievement hunters that exist since achievements were first released on Steam in 2007.
+[100Pals/AchievementHunting.com](https://achievementhunting.com/) (Discord Partner: https://discord.gg/100Pals) is an all-platforms achievement hunting community founded in 2015.
```
Comment (wescopeland) [inline docs/guidelines/users/global-leaderboard-and-achievement-hunting-rules.md]:
```suggestion
[100Pals/AchievementHunting.com](https://achievementhunting.com/) is an all-platforms achievement hunting community founded in 2015.
```
Happy to provide a backlink, but let's keep it to one domain. Going to your dot-com will provide the most SEO juice, if that helps.

**Review by wescopeland — COMMENTED** [review]
Hi! Thanks for opening a PR to correct some of these details. I have some minor feedback below.

**Comment by Xeinok-Pals** [comment]
sounds good, thanks

**Review by wescopeland — APPROVED** [review]

**Pull request merged**